wp-content/themes/pub/wporg-learn-2020/css/print.css

```diff
@@ -157,5 +157,3 @@ p a {
 .single-wporg_workshop .workshop-page_content {
   margin-bottom: 32px;
 }
-
-/*# sourceMappingURL=print.css.map */
```

Comment:
Why remove the map?

Reply:
It's build function. At PR merge i think map will not show on live. It only for dev. I restored for now.